### PR TITLE
Removes Ligma, changes some disease names.

### PR DESCRIPTION
**Pull request by MarinaGryphon**
* Removes the Ligma disease.
* Changes a few disease names up. Space Cold to Rhinovirus, Space Flu to Influenza.

### CODE REVIEW

**Comment by OneOneThreeEight** [comment]
https://youtu.be/dnRTt34wAhk Giggle seizures are a thing. Add it back but not named Ligma.

pls.

**Comment by BurgerLUA** [comment]
make an ic changelog and an ooc changelog. both must be >1000 words

File: code/modules/virus2/generated_diseases.dm [hunk 1]
```diff
@@ -100,19 +100,6 @@
 		/datum/disease2/effect/bones
 	)
 
-/datum/disease2/disease/generated/laughter
```
Comment (LordFowl) [inline code/modules/virus2/generated_diseases.dm]: The disease itself is still legitimate, just not the name.

**Comment by LordFowl** [comment]
That's literally never gonna happen but ok, what do I know

**Pull request merged**